voluncheer/static/js/main.js

```diff
@@ -27,4 +27,17 @@ function ShowDetail(element) {
   } else {
     form.style.display = "none";
   }
+};
+
+function ShowSibling(element) {
+  var id = element.id;
+  var form = document.getElementById(id+"-siblings");
+  console.log(form.style.display);
```

Comment:
we can remove this` console.log`, which I assume was for debugging

Reply:
THanks for pointing out!!